presto-benchto-benchmarks/pom.xml

```diff
@@ -51,6 +51,10 @@
                     <groupId>org.hibernate</groupId>
                     <artifactId>hibernate-validator</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
In my opinion, we should avoid excluding dependencies. Instead, we can enforce using a CVE-free version of this dependency through Dependency Management.

It is possible this dependency might be required when running benchto which is not detected in our CI pipeline.

Reply:
+1

Reply:
why do we have so many exclusions from benchto-driver? We control the project, and it's entirely for Presto. Let's instead fix the dependencies in Benchto (for all these exclusions) https://github.com/prestodb/benchto, and then upgrade our benchto version.  When we upgrade, the dependency groupId will change from com.teradata to io.prestodb.